### DISCUSSION TITLE
Support windows.nvim

### DISCUSSION THREAD

**Discussion by radvil** [folke/edgy.nvim#22] [Q&A]
Hi @folke, always great plugins btw. Any chance to support this https://github.com/anuvyklack/windows.nvim ? 
Seems like it broke when option autoresize on windows nvim activated. I am a newbie, I hope I find a good solution fot that.
This happens only when firing edgy.toggle() command, but if I trigger it on Neotree, it seems fine.

**Comment by folke** [comment]
I'm actually using `windows.nvim` myself, so it should work. I did however disable it's animations and I use `mini.animate` instead, which plays nice with edgy.

You could also just disable edgy animations, which should work.
